ethcore/call-contract/src/call_contract.rs

```diff
@@ -16,14 +16,70 @@
 
 //! Provides CallContract and RegistryInfo traits
 
+use std::fmt;
 use bytes::Bytes;
-use ethereum_types::Address;
+use derive_builder::Builder;
+use ethereum_types::{Address, U256};
 use types::ids::BlockId;
 
+#[derive(Debug)]
+/// Transaction call error
+pub enum CallError {
+	/// Call reverted
+	Reverted(String),
+	/// Other error
+	Other(String),
+}
+
+impl fmt::Display for CallError {
+	fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+		fmt::Debug::fmt(&self, f)
+	}
+}
+
+#[derive(Builder, Default, Clone)]
+#[builder(default)]
```

Comment:
I believe this using this macro once is not worth bringing 5 new external dependencies to the project

Reply:
I agree, that using this one once does not make really sense! That and `CallError` have been introduced by @tomusdrw suggestion in [this](https://github.com/paritytech/parity-ethereum/pull/10602#discussion_r294176475) comment. I understood that intention to use builder was because it was planned to be used in other places. And according to `CallError` usage I assumed this one was suggested as the base boilerplate and is gonna be extended in the future for other cases which actually depend on the error type. My bad - I didn't clarify that before implementation, and I don't mind getting rid of this (I think it's not a big deal with builder macro, but more unsure about `CallError`), but I would like first to clarify this in order to prevent reverting the changes back and forth.

Reply:
@tomusdrw is there any reason to have 2 separate enum variants for `CallError`? We don't use them at all

Also, do you think we should commit to `derive_builder` and if yes, should we also use it in other places?